frontend/packages/pipelines-plugin/src/plugin.tsx

```diff
@@ -470,10 +470,11 @@ const plugin: Plugin<ConsumedExtensions> = [
     properties: {
       id: 'pipeline',
       url: `/k8s/ns/:namespace/${referenceForModel(PipelineModel)}/~new/builder`,
-      label: 'Pipeline',
+      // t('pipelines-plugin~Pipeline')
+      label: '%pipelines-plugin~Pipeline%',
       // t('pipelines-plugin~Create a Tekton Pipeline to automate delivery of your Application')
       description:
-        '%pipelines-plugin~Create a Tekton Pipeline to automate delivery of your application%',
+        '%pipelines-plugin~Create a Tekton Pipeline to automate delivery of your Application%',
```

Comment:
@beaumorley should `application` be lower case or capitalized in the console?

Reply:
@christianvogt @vikram-raj capitalizing it matches other places in the console so I would go with that.